MPChartLib/src/main/java/com/github/mikephil/charting/highlight/PieRadarHighlighter.java

```diff
@@ -32,7 +34,7 @@ public Highlight getHighlight(float x, float y) {
         PieDataSet dataSet = ((PieDataSet)mChart.getData().getDataSetByIndex(0));
         float extraDistance = 0.0f;
         if ( dataSet.isDrawValueTextBubbleEnabled()) {
-            extraDistance = dataSet.getSelectionShift() + dataSet.getValueTextBubbleSpacing()*1.5f;
+            if (extraSelectionDistance == 0) extraSelectionDistance = Utils.calcTextWidth(mChart.getRenderer().getPaintValues(),"100.0%") + dataSet.getSelectionShift() + dataSet.getValueTextBubbleSpacing();
```

Comment:
Maximum text width we could run into might be more than `100.0%` depending on locale. I might be wrong but in some cases it could be `100.0[ ]%`.

Reply:
yeah I can give it a bit more to be on the safe side. No harm if it's a bit bigger

Reply:
updated

Reply:
👍